src/arch/arm/mmu/cache.c

```diff
@@ -8,14 +8,40 @@
 
 #include <stddef.h>
 #include <stdint.h>
+#include <inttypes.h>
+#include <util/log.h>
+
+static uint32_t get_cache_line_size(void) {
+#if __ARM_ARCH == 6 || __ARM_ARCH == 5
+	/*
+	 * I am not sure about if it is arch defined cache line size,
+	 * but Linux uses 32 bit cache line size for both I and D caches
+	 * for any ARMV6: arch/arm/mm/cache-v6.S
+	 *
+	 * For ARMv5 I am not sure at all, just add this to fix arm/qemu.
+	 *
+	 * P.S. It is added because of ARM1176JZF-S (Ra Pi model B rev 2.0)
+	 */
+	return 32;
```

Comment:
Is it 32 bits or 32 bytes? `line_size` seems to be inerpreted as bytes count in `dcache_inval()`, but in comments it's `32 bit cache line size`

Reply:
32 bytes, it's a typo. Thank you.

Reply:
Fixed.